x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/ilm/IndexLifecycle.java

```diff
@@ -330,11 +329,7 @@ private static List<NamedXContentRegistry.Entry> xContentEntries() {
         );
 
         // TSDB Downsampling / Rollup
-        if (IndexSettings.isTimeSeriesModeEnabled()) {
-            entries.add(
-                new NamedXContentRegistry.Entry(LifecycleAction.class, new ParseField(RollupILMAction.NAME), RollupILMAction::parse)
-            );
-        }
+        entries.add(new NamedXContentRegistry.Entry(LifecycleAction.class, new ParseField(RollupILMAction.NAME), RollupILMAction::parse));
```

Comment:
Please remove `LifecycleAction` completely from here and I will enable it in https://github.com/elastic/elasticsearch/pull/87269